src/opm/parser/eclipse/EclipseState/Schedule/Network/Balance.cpp

```diff
@@ -60,15 +61,28 @@ Balance::Balance(const Tuning& tuning, const DeckKeyword& keyword) {
         this->m_min_tstep = record.getItem<NB::MIN_TIME_STEP>().getSIDouble(0);
 }
 
+namespace {
 
-Balance::Balance(bool network_active, const Tuning& tuning)
+double get_si_default(const NB& parser_keyword, const std::string& item_name, UnitSystem& unit_system) {
+    const auto& item = parser_keyword.getRecord(0).get(item_name);
+    return unit_system.to_si( item.dimensions()[0], item.getDefault<double>());
```

Comment:
Does `item.getDefault<>()` return a value that depends on the `UnitSystem`?  If not, I don't understand how this will do the right thing in unit conventions other than `METRIC`.

Reply:
> If not, I don't understand how this will do the right thing in unit conventions other than METRIC.

Arrghh - you are right.